harvard-lts/drs-translation-service
# Unit tests for ETD packages of various kinds

**Unit tests for translating the data structure and building batches.**
* * *

**JIRA Issue**: https://at-harvard.atlassian.net/browse/ETD-307

# What does this Pull Request do?
Adds unit tests for various ETD package types

# How should this be tested?

Run pytests (see [README](https://github.com/harvard-lts/drs-translation-service#testing))

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? yes
- integration tests? yes (in etd-integration-tests)

# Interested parties
@awoods
